Update database-persistence.md (#3199)
RedisLabs/redislabs-docs: content/rs/databases/configure/database-persistence.md

## Auto Tiering data persistence

> Auto Tiering Flash storage is not considered as persistent storage.

Reviewer: ```suggestion
Auto Tiering flash storage is not considered persistent storage.
```

> Flash-based databases are expected to hold larger datasets and repair
> times for shards can be longer under node failures. To better protect
> the database against node failures, users might consider enabling
> master and replica dual data persistence provides better protection
> against failures under these longer repair times.

Reviewer: ```suggestion
Flash-based databases are expected to hold larger datasets, and shard repair times can take longer after node failures. To better protect the database against node failures with longer repair times, consider enabling master and replica dual data persistence.
```

However, dual data persistence with replication adds some processor
and network overhead, especially for cloud configurations
with network-attached persistent storage, such as EBS-backed
volumes in AWS.

There may be times when performance is critical for your use case and
you don't want to risk data persistence adding latency.

> You can enable or disable data-persistence on the master shards using the

Reviewer: ```suggestion
You can enable or turn off data persistence on the master shards using the
```
